src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -967,6 +967,53 @@ describe('Repeater item component', () => {
 
       flushDropdownTimer();
     }));
+
+    it('should emit the isSelectedChange event when the user selects an item', fakeAsync(() => {
```

Comment:
@michael-tims Thanks for writing unit tests! It looks like we already have a test for `should select item with space and enter keys when selectable is set to true`. Instead of writing a whole new test just to test the `isSelectedChange` - could we just add a line to the preexisting test that checks to make sure the model is updated correctly?

Reply:
Also, we want to make sure that the emitter is being called with the correct value (`true`/`false`).

Reply:
Done